pxt.json

```diff
@@ -8,13 +8,23 @@
     },
     "files": [
         "README.md",
-        "seesaw.ts"
+        "seesaw.ts",
+        "analoginput.ts",
+        "crickit.ts",
+        "digitalio.ts",
+        "pwmout.ts",
+        "samd09.ts"
     ],
     "testFiles": [
         "test.ts"
     ],
-    "testDependencies": {
-        "circuit-playground": "*"
+    "public": false,
+    "targetVersions": {
```

Comment:
I need to fix this in our CLI, it should not be modifying pxt.json. That one is on me.